Shared/Samples/Browse OGC API feature service/BrowseOGCAPIFeatureServiceView.swift

```diff
@@ -0,0 +1,256 @@
+                                    }
+                                }
+                            }
+                            .pickerStyle(.automatic)
+                        }
+                    }
+                }
+                .alert("Load OGC API feature service", isPresented: $presentAlert, actions: {
+                    TextField("URL:", text: $userInput)
+                    Button("Load", action: {
+                        presentAlert = false
+                        isLoading = true
+                        Task {
+                            do {
+                                try await model.loadOGCFeatureData(
+                                    mapProxy: mapProxy,
+                                    url: URL(string: userInput)
+                                )
+                                selection = model.layerNames.first ?? ""
+                            } catch {
+                                self.error = error
+                            }
+                        }
+                    })
+                    Button("Cancel", role: .cancel, action: {
+                        presentAlert = false
+                    })
+                }, message: {
+                    Text("Please provide a URL to an OGC API feature service.")
+                })
+                .errorAlert(presentingError: $error)
+        }
+    }
+}
+
+private extension BrowseOGCAPIFeatureServiceView {
+    @MainActor
+    class Model: ObservableObject {
+        let map: Map = {
+            let map = Map(basemapStyle: .arcGISTopographic)
+            map.initialViewpoint = Viewpoint(
+                center: Point(
+                    latitude: 32.62,
+                    longitude: 36.10
+                ),
+                scale: 200000
+            )
+            return map
+        }()
+        
+        @Published var layerNames: [String] = []
+        
+        private var featureCollectionInfos: [OGCFeatureCollectionInfo] = []
+        private var service: OGCFeatureService!
+        var selectedInfo: OGCFeatureCollectionInfo?
+        
+        /// The query parameters to populate features from the OGC API service.
+        private let queryParameters: QueryParameters = {
+            let queryParameters = QueryParameters()
+            // Set a limit of 1000 on the number of returned features per request,
+            // because the default on some services could be as low as 10.
+            queryParameters.maxFeatures = 1_000
+            return queryParameters
+        }()
+        
+        /// Returns a renderer for a specified geometry type.
+        /// - Parameter geometryType: The ARCGis Geometry type.
+        /// - Returns: Returns a `SimpleRenderer` optional with the correct settings for the given geometry.
+        private func getRendererForTable(withType geometryType: Geometry.Type) -> SimpleRenderer? {
+            var renderer: SimpleRenderer?
+            if geometryType == Point.self {
```

Comment:
Trying to come up with a clean way of switching through all the geometry types  and setting the renderer.

Reply:
Nope, Geometry type is not equatable so this seems like the best method.

Reply:
You can still use `switch-case` here. Such as

```swift
switch geometryType {
case is Point.Type:
    print("Point!")
default:
    print("Not Point.")
}
```